docs/csharp/tutorials/ranges-indexes.md

```diff
@@ -6,7 +6,7 @@ ms.custom: mvc
 ---
 # Indices and ranges
 
-Ranges and indices provide a succinct syntax for accessing single elements or ranges in an <xref:System.Array>, <xref:System.Span%601>, or <xref:System.ReadOnlySpan%601>. These features enable more concise, clear syntax to access single elements or ranges of elements in a sequence.
+Ranges and indices provide a succinct syntax for accessing single elements or ranges in an <xref:System.Array>, <xref:System.String>, <xref:System.Span%601>, or <xref:System.ReadOnlySpan%601>. These features enable more concise, clear syntax to access single elements or ranges of elements in a sequence.
```

Comment:
Added `string` to the list in order to partially address #14449

Reply:
I like this. 

Is this a good place to describe `Slice`, and other elements of [implicit range support](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/index-range-changes.md#implicit-range-support)? I'm not convinced either way yet. What are your thoughts?

Reply:
[This tutorial](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/ranges-indexes) might be a good place to include the section about the implicit range support because that page, most probably, is the main source of information about indices and ranges in the docset for now. Though it would result in the reference material being presented in a tutorial. However, maybe, for the feature adoption/introduction period, this is good to have all the information as dense as possible. Later, in several months, the reference material might be moved to the reference section of the docs, though. So, I'm on a fence here, but to get started, I would update the tutorial page.

Reply:
@BillWagner do we resolve it for now as to update the tutorial as the fix for #14449? What do you think?

Reply:
Yes, I like that as a plan.